CONTRIBUTING.md

```diff
@@ -115,4 +115,12 @@ $ node_modules/.bin/gulp serve --server=<che_api_url>
 
 
 ### Che server a.k.a WS master
-To be provided soon.
+For now Che in Che flow is not completed.
+However one may build Che server inside Che.
```

Comment:
I don't think we should say that `the flow is not completed` ... just say what you can do.
- fast build
- start che server ? -> access to swagger ? call/test few api like creating a workspace maybe ? etc ... run that with a real database ?

Reply:
Build (with some limitations in tests) is the only thing we can do now...